src/_data/pkg-vers.json

```diff
@@ -8,6 +8,7 @@
     "doc-path": "components",
     "prev-vers": "0.8.0",
     "vers": "0.13.0+1",
+    "doc_vers": "0.13.0",
```

Comment:
You've added `doc_vers` to `angular_components` but aren't using it, was this by mistake?

As I've mentioned elsewhere, you don't need this new field, just update `vers` instead.